khal/parse_datetime.py

```diff
@@ -413,12 +413,20 @@ def guessrangefstr(daterange, locale,
     )
 
 
-def rrulefstr(repeat, until, locale):
+def rrulefstr(repeat, until, every, locale):
     if repeat in ["daily", "weekly", "monthly", "yearly"]:
         rrule_settings = {'freq': repeat}
         if until:
             until_dt, is_date = guessdatetimefstr(until.split(' '), locale)
             rrule_settings['until'] = until_dt
+        if every:
+            try:
+                int(every)
+            except ValueError:
+                logger.fatal("Invalid value for every option. " +
+                             "Must be an integer")
```

Comment:
The "+" isn't required here. Quotes around `every` would make things clearer, to distinguish from "every option must be an integer".
To copy from below:
```suggestion
                logger.fatal("Invalid value for the 'every' option. "
                             "Must be an integer")
```